crowdin.yml

```diff
@@ -17,6 +17,12 @@ files:
   - source: /jekyll/_includes/snippets/*.adoc
     translation: /jekyll/_includes/snippets/%two_letters_code%/%original_file_name%
     update_option: update_as_unapproved
+  - source: /jekyll/_includes/snippets/faq/*.adoc
+    translation: /jekyll/_includes/snippets/%two_letters_code%/%original_file_name%
+    update_option: update_as_unapproved
+  - source: /jekyll/_includes/snippets/troubleshoot/*.adoc
+    translation: /jekyll/_includes/snippets/%two_letters_code%/%original_file_name%
```

Comment:
Same comment here, I would expect a `troubleshoot` directory in the `translation:` line path.